cumulusci/core/tasks.py

```diff
@@ -88,6 +90,16 @@ def _init_options(self, kwargs):
         if kwargs:
             self.options.update(kwargs)
 
+        # Handle default values
+        for name, config in list(self.task_options.items()):
+            if name not in self.options and "default" in config:
+                self.options[name] = config["default"]
+
+            if "do_replacement" in config:
+                self.options[name] = self.options[name].format(
+                    **{"project_config": self.project_config}
+                )
```

Comment:
In the long run I don't want to have 2 different ways to do variable interpolation into option values (this one, and the one just below).

This also assumes every option name is in self.options and every value in self.options is a string, both of which are invalid.

Reply:
The old way of doing templates seems pretty limiting so it should be removed someday, but that's a backwards incompatible change so it will take some time.